x-pack/metricbeat/module/mssql/_meta/docs.asciidoc

```diff
@@ -0,0 +1,2 @@
+This is the mssql module.
```

Comment:
I think the module description could be a bit more extensive.

Reply:
Done :)